blueprints/structural_sections/concrete/reinforced_concrete_sections/plotters/rectangular.py

```diff
@@ -0,0 +1,408 @@
+            horizontalalignment="center",
+            arrowprops=diameter_line_style,
+            rotation=90,
+            annotation_clip=False,
+        )
+        self.axes[axes_i].text(
+            s=custom_text_height or f"{self.cross_section.height:.0f} mm",
+            x=offset_height,
+            y=0,
+            verticalalignment="center",
+            horizontalalignment="right",
+            fontsize=font_size_dimension,
+            rotation=90,
+        )
+
+    def _add_stirrups(
+        self,
+        axes_i: int = 0,
+    ) -> None:
+        """Adds stirrups to the plot.
+
+        Parameters
+        ----------
+        axes_i: int
+            Index of the axes to plot on. Default is 0.
+        """
+        for stirrup in self.cross_section.stirrups:
+            left_bottom = Point(stirrup.geometry.exterior.coords[0])  # left bottom point of the stirrup (center line)
+            self.axes[axes_i].add_patch(
+                mplpatches.Rectangle(
+                    xy=(left_bottom.x - stirrup.diameter / 2, left_bottom.y - stirrup.diameter / 2),
+                    width=stirrup.ctc_distance_legs + stirrup.diameter,
+                    height=self.cross_section.height - self.cross_section.covers.upper - self.cross_section.covers.lower,
+                    facecolor=STIRRUP_COLOR,
+                    fill=True,
+                )
+            )
+            self.axes[axes_i].add_patch(
+                mplpatches.Rectangle(
+                    xy=(left_bottom.x + stirrup.diameter / 2, left_bottom.y + stirrup.diameter / 2),
+                    width=stirrup.ctc_distance_legs - stirrup.diameter,
+                    height=self.cross_section.height - self.cross_section.covers.upper - self.cross_section.covers.lower - 2 * stirrup.diameter,
+                    facecolor=RCS_CROSS_SECTION_COLOR,
+                    fill=True,
+                )
+            )
+
+    def _add_longitudinal_rebars(
+        self,
+        axes_i: int = 0,
+    ) -> None:
+        """Adds longitudinal rebars to the plot.
+
+        Parameters
+        ----------
+        axes_i: int
+            Index of the axes to plot on. Default is 0.
+        """
+        for rebar in self.cross_section.longitudinal_rebars:
+            self.axes[axes_i].add_patch(
+                mplpatches.Circle(
+                    xy=(rebar.x, rebar.y),
+                    radius=rebar.radius,
+                    linewidth=1,
+                    color=REBAR_COLOR,
+                )
+            )
+
+    def _legend_text(self) -> str:
+        """Creates the legend text.
+
+        Returns
+        -------
+        str
+            Legend text.
+        """
+        # start building legend
+        main_steel_material_used = self.cross_section.get_present_steel_materials()[0].name
+        legend_text = f"{self.cross_section.concrete_material.concrete_class.value} - {main_steel_material_used}"
+
+        legend_text += self._add_stirrups_to_legend()
```

Comment:
i could make it a public method perhaps. Also notice that this is only the standard plotter of the Blueprints RCS. It can always be replaced by a customized plotter where custom changes can be introduced. For example:

```python
...

# Define a concrete material
concrete = ConcreteMaterial(concrete_class=ConcreteStrengthClass.C35_45)

# Define a rectangular reinforced cross-section
cs = RectangularReinforcedCrossSection(
    width=1000,
    height=800,
    covers=(45, 30, 35, 50),  # upper, right, lower, left
    concrete_material=concrete,
)

# change to custom company plotter
cs.plotter = SomeRandomFancyCompanyPlotter(cross_section=cs)

...